patterns/sidebar-writer.php

```diff
@@ -17,7 +17,7 @@
 <!-- /wp:heading -->
 
 <!-- wp:paragraph {"style":{"typography":{"fontSize":"0.9rem"}}} -->
-<p style="font-size:0.9rem"><?php echo esc_html_x( 'Je suis Adele Diouf, et je suis animée par une curiosité insatiable de dévoiler la tapisserie complexe de la finance et de l\'économie. Au cours de plus d\'une décennie, mon voyage à travers le labyrinthe des marchés mondiaux, des politiques monétaires et des tendances économiques a fourni aux lecteurs les outils nécessaires pour naviguer dans le paysage dynamique de la finance.', 'sample content for sidebar-writer', 'twentytwentyfour' ); ?></p>
+<p style="font-size:0.9rem"><?php echo esc_html_x( 'I\'m Adele Diouf, and I\'m driven by an insatiable curiosity to unravel the intricate tapestry of finance and economics. Over the course of more than a decade, my journey through the maze of global markets, monetary policies, and economic trends has provided readers with the tools necessary to navigate the dynamic landscape of finance.', 'sample content for sidebar-writer', 'twentytwentyfour' ); ?></p>
```

Comment:
Actually, since this text is about the post author or site owner, I think this should use the post author biography block; what do you think?

Reply:
something Like this instead of paragraph ?

<!-- wp:post-author-biography /-->

Reply:
@carolinan 
I have added post author biography block for that

Reply:
Please test the pattern carefully. A block can not be placed **inside** a paragraph block.

Reply:
@carolinan 

I have updated changes. let me know if any correction.